### PR TITLE
[Azure Confidential Ledger] GA CLI extension

### PR DESCRIPTION
**Pull request by lynshi**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
image

- [x] Have you run `python scripts/ci/test_index.py -q` locally?
image

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Confidential Ledger

File: src/confidentialledger/setup.py [hunk 1]
```diff
@@ -37,22 +37,22 @@
...
-    description='Microsoft Azure Command-Line Tools ConfidentialLedger Extension',
-    author='Microsoft Corporation',
-    author_email='azpycli@microsoft.com',
-    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/confidentialledger',
-    long_description=README + '\n\n' + HISTORY,
-    license='MIT',
+    description="Microsoft Azure Command-Line Tools ConfidentialLedger Extension",
+    author="Microsoft Corporation",
+    author_email="azpycli@microsoft.com",
+    url="https://github.com/Azure/azure-cli-extensions/tree/master/src/confidentialledger",
```
Comment (kairu-ms) [inline src/confidentialledger/setup.py:51]:
```suggestion
    url="https://github.com/Azure/azure-cli-extensions/tree/main/src/confidentialledger",
```

**Comment by kairu-ms** [comment]
Please fix linter issues. Thanks.

**Comment by lynshi** [comment]
> Please fix linter issues. Thanks.

Done - sorry about that, the issues weren't uncovered by the commands in the PR template.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**